kategory/src/main/kotlin/kategory/data/Option.kt

```diff
@@ -136,8 +137,9 @@ sealed class Option<out A> : OptionKind<A> {
     /**
      * Returns false if the option is $none, true otherwise.
      * @note Implemented here to avoid the implicit conversion to Iterable.
```

Comment:
Scala garbage in comments :fire: I think we should remove all these comments form the original ports in all datatypes since our implementations have by now deviated a lot.

Reply:
I'm not a fan of porting the original documentation either. cc @JorgeCastilloPrz to comment it in, as he's more anal about inline documentation.

Reply:
Let's address this in other PR.